README.md

```diff
@@ -51,27 +51,27 @@ class app_widgetContainer {
 }
 ```
 
-For methods, follow the standard Apple convention of referring to the first parameter in the method name:
+For methods, when the first argument is part of a prepositional phrase, make the preposition the first label and use the actual argument name inside the method definition:
 
 ```swift
 class Guideline {
-  func combineWithString(incoming: String, options: Dictionary?) { ... }
-  func upvoteBy(amount: Int) { ... }
+  func combine(with incoming: String, options: Dictionary?) { ... }
+  func upvote(by amount: Int) { ... }
```

Comment:
I think that this case still fits here. The signature is optimized for the call site:
item(with: customer.name)
vs.
item(withName: customer.name)

I just did a search for something like this in the UPPAbaby codebase, and we do this all the time. It reads well; also I don't remember any problems using the APIs.

It would be great if Xcode used "name" when doing code completion. Right now it doesn't,; maybe some day :–)  Most importantly to me is that this is consistent with the "grand renaming". If we don't do this, our code will stand out from UIKit / Swift StdLib APIs.